src/__tests__/migration-tester.ts

```diff
@@ -0,0 +1,108 @@
+// Note: the is a helper function to test migrations by postgres-shift.
+// @see https://github.com/porsager/postgres-shift/blob/e7a4cb16d66c33c73f4f730b7d1de33b7757fea9/index.js
```

Comment:
I did consider moving the code of the package into our codebase but this can be done in a separate PR. I have a local implementation stashed that migrates it and this test function into a singular class. What's our opinion? Should we opt for that instead?